openpype/hosts/flame/api/plugin.py

```diff
@@ -710,19 +709,36 @@ def __init__(self, openclip_file_path, feed_data):
         self.feed_dir = os.path.dirname(feed_path)
         self.feed_ext = os.path.splitext(self.feed_basename)[1][1:].lower()
 
-        if not os.path.isfile(openclip_file_path):
+        if not self._is_valid_tmp_file(openclip_file_path):
             # openclip does not exist yet and will be created
             self.tmp_file = self.out_file = openclip_file_path
             self.create_new_clip = True
 
         else:
+            # update already created clip
             # output a temp file
             self.out_file = openclip_file_path
             self.tmp_file = os.path.join(self.feed_dir, self.tmp_name)
+
+            # remove previously generated temp files
+            # it will be regenerated
             self._clear_tmp_file()
 
         self.log.info("Temp File: {}".format(self.tmp_file))
 
+    def _is_valid_tmp_file(self, file):
+        # check if file exists
+        if os.path.isfile(file):
+            # test also if file is not empty
+            with open(file) as f:
+                lines = f.readlines()
+                if len(lines) > 2:
+                    return True
+
+                # file is probably corrupted
+                os.remove(file)
+                return False
```

Comment:
You're removing opened file.
```suggestion
            with open(file) as f:
                lines = f.readlines()
            if len(lines) > 2:
                return True

            # file is probably corrupted
            os.remove(file)
            return False
```